workers/unity/Packages/com.improbable.gdk.core/Config/LocatorConfig.cs

```diff
@@ -55,16 +74,51 @@ public void SetProjectName(string projectName)
         }
 
         /// <summary>
-        ///     Sets the login token for the Locator to use.
+        ///     Sets the LocatorCredentialsType to LoginToken and sets the login token for the Locator to use.
         /// </summary>
+        /// <remarks>
+        ///     SetLoginToken() and SetSteamCredentials() may not be called simultaneously when connecting to a deployment.
+        /// </remarks>
         /// <param name="loginToken">The login token.</param>
+        /// <exception cref="ConnectionFailedException">
+        ///     Thrown if SetLoginToken() is called after SetSteamCredentials() was called before.
+        /// </exception>
         public void SetLoginToken(string loginToken)
         {
+            if (LocatorParameters.CredentialsType == LocatorCredentialsType.Steam)
+            {
+                throw new ConnectionFailedException(
+                    "SetLoginToken() and SetSteamCredentials() may not be called simultaneously when connecting to a deployment.",
```

Comment:
I don't think this should reference function names with incorrect signatures.

Reply:
Possibly better to just be descriptive here. Maybe "Could not set a SpatailOS login token because Steam credentials have already been set. Only one can be used when connecting to a deployment"

Alternatively we can just switch them. Although I think I prefer it throwing an error like this.

Reply:
Rephrased the sentences and made them more descriptive.